### PR TITLE
[CNSMR-1523] Add framework creation documentation

### PR DESCRIPTION
**Pull request by AliSoftware**
As suggested by @mr-v in related PRs

Updated instructions to take into account the changes from CNSMR-1511

### CODE REVIEW

**Review by inamiy — APPROVED** [review]
👍

**Review by mr-v — APPROVED** [review]
Thank you very much!

File: Cookbook/Technical-Documents/AddingFrameworks.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+   * `BabylonNewFramework.release.xcconfig`:
+
+	```
+	#include "../BuildConfigs/Framework.Release.xcconfig"
+	#include "../Pods/Target Support Files/Pods-BabylonNewFramework/Pods-BabylonNewFramework.release.xcconfig
+	```
+
+3. Add the `BuildConfigs` folder to your Xcode project (by drag & drop or right-click + "Add Files to ...").
+   * Be extra sure NOT to include the `xcconfig` files in the target when adding the references (the `xcconfig` files should be referenced in the project but should not belong in any target so **uncheck all target checkboxes**)
```
Comment (mr-v) [inline Cookbook/Technical-Documents/AddingFrameworks.md]: > (the `xcconfig` files should be referenced in the project but should not belong in any target so **uncheck all target checkboxes**)

Idea for the projects' configuration - maybe we could `xcconfig` files to `EXCLUDED_SOURCE_FILE_NAMES` build setting to be extra sure about it.

Comment (AliSoftware) [reply Cookbook/Technical-Documents/AddingFrameworks.md]: Oh... I had the option on my list for Danger rule ideas to make Danger warn about those, but `EXCLUDED_SOURCE_FILE_NAMES` could be a nice solution/alternative indeed!

**Pull request merged**